mocha/index.d.ts

```diff
@@ -110,6 +106,24 @@ declare class Mocha {
 
 // merge the Mocha class declaration with a module
 declare namespace Mocha {
+    interface ISuiteCallbackContext {
```

Comment:
:+1: for sticking w/ the convention of the file & starting these interfaces with an I